Source/Internal/IGListAdapter+DebugDescription.m

```diff
@@ -32,6 +32,7 @@ - (NSString *)debugDescription {
     [debug addObject:[NSString stringWithFormat:@"Scroll view delegate: %@", self.scrollViewDelegate]];
     [debug addObject:[NSString stringWithFormat:@"Is in update block: %@", IGListDebugBOOL(self.isInUpdateBlock)]];
     [debug addObject:[NSString stringWithFormat:@"View controller: %@", self.viewController]];
+    [debug addObject:[NSString stringWithFormat:@"Is in update block: %@", IGListDebugBOOL(self.collectionView.isPrefetchingEnabled)]];
```

Comment:
Should probably be worded something like "Is Prefetching Enabled:"

Bit of copypasta 😉